scripts/zowe-explorer-api-configure.sh

```diff
@@ -60,4 +58,41 @@ chmod 755 $ZOWE_ROOT_DIR/$EXPLORER_INSTALL_FOLDER/scripts
 # Configure explorer jobs api ended
 #############################################
 
+#############################################
```

Comment:
Post 1.0 we need to clean up this file (and probably a bunch of others) to refactor and remove all the copy-pasting.

Reply:
New PR https://github.com/zowe/zowe-install-packaging/pull/248 created to optimize the install/configure scripts.